sycl/unittests/accessor/AccessorIterator.cpp

```diff
@@ -277,6 +277,24 @@ TEST_F(AccessorIteratorTest, LegacyRandomAccessIteratorRequirementsExtra) {
     ASSERT_FALSE(It3 < It);
     ASSERT_FALSE(It == It3);
   }
+
+  { // It - n equivalent to:
+    // iterator temp = It;
+    // return temp -= n;
+    auto It1 = accessor.end();
+    auto It2 = accessor.end();
+
+    It2 -= 3;
+    ASSERT_EQ(It1 - 3, It2);
+    ASSERT_EQ(It1, accessor.end());
+  }
+  {
+    const auto It1 = accessor.end();
```

Comment:
Let's leave a comment that here we test `it - n` where `it` is const. Perhaps we could even merge this into the section about which you added, just as `const auto It3 = ...; ASSERT_EQ(It3 - 3, It2);`

Reply:
This one and the comment above - done 1aa5d7f